src/reanimated2/core.js

```diff
@@ -234,6 +235,16 @@ export const runOnJS = (fun) => {
   }
 };
 
+export function createAnimatedPropAdapter(adapter, nativeProps) {
```

Comment:
Why do we need `adapter` here? It's not used and just returned.

Reply:
That's how it's designed, I wanted to create one place where adapter and native props would be passed, plus, look at the plugin, it does automatic workletization of the adapter.